xml/schema/Contribute/Contribution.xml

```diff
@@ -309,6 +309,11 @@
       <type>Text</type>
     </html>
   </field>
+  <index>
+    <name>source</name>
```

Comment:
When I run createIndexes() locally, it makes an index with name `index_source`.

To match, should the XML say `<name>index_source</name>`